.gitignore

```diff
@@ -3,4 +3,5 @@ target/
 *.iml
 __pycache__/
 
-.flattened-pom.xml
\ No newline at end of file
+.flattened-pom.xml
+*.patch
```

Comment:
was this change intentional?

Reply:
I was maintaining multiple patches in my local, so thought it would be useful. Do you want me to remove this ?

Reply:
Fixed as part of https://github.com/googleapis/java-datastore/pull/823/commits/8c81b9f201102b4db79d195263d15fb06e4985d3.